packages/consumption/test/modules/requests/itemProcessors/shareAttribute/ShareAttributeRequestItemProcessor.test.ts

```diff
@@ -138,27 +138,32 @@ describe("ShareAttributeRequestItemProcessor", function () {
             const sender = testAccount.identity.address;
             const recipient = CoreAddress.from("Recipient");
 
+            // eslint-disable-next-line jest/no-conditional-in-test
```

Comment:
@jkoenig134 What's our approach to those linter warnings?

Reply:
not have them :D

Reply:
@jkoenig134 So is it fine as it is, I can resolve the conversation and approve the PR?

Reply:
no, please remove that comment.

Reply:
it just removes a warning, but we want to have this warning.